addons/website_hr_recruitment/static/src/js/website_hr_applicant_form.js

```diff
@@ -15,28 +15,29 @@ publicWidget.registry.hrRecruitment = publicWidget.Widget.extend({
     },
 
     _onClickApplyButton (ev) {
-        const $linkedin_profile = $('#recruitment4');
-        const $resume = $('#recruitment6');
+        const linkedinProfileEl = document.querySelector("#recruitment4");
+        const resumeEl = document.querySelector("#recruitment6");
 
-        const is_linkedin_empty = !$linkedin_profile.length || $linkedin_profile.val().trim() === '';
-        const is_resume_empty = !$resume.length || !$resume[0].files.length;
-        if (is_linkedin_empty && is_resume_empty) {
-            $linkedin_profile.attr('required', true);
-            $resume.attr('required', true);
+        const isLinkedinEmpty = !linkedinProfileEl || linkedinProfileEl.value.trim() === "";
+        const isResumeEmpty = !resumeEl || !resumeEl.files.length;
+        if (isLinkedinEmpty && isResumeEmpty) {
+            linkedinProfileEl?.setAttribute("required", true);
+            resumeEl?.setAttribute("required", true);
         } else {
-            $linkedin_profile.attr('required', false);
-            $resume.attr('required', false);
+            linkedinProfileEl?.removeAttribute("required");
+            resumeEl?.removeAttribute("required");
         }
     },
 
     hideWarningMessage(target, messageContainerId) {
-        $(target).removeClass("border-warning");
-        $(messageContainerId).hide();
+        target.classList.remove("border-warning");
+        document.querySelector(messageContainerId).classList.add("d-none");
     },
 
     ShowWarningMessage(target, messageContainerId, message) {
```

Comment:
While we are at it, what do you think about also fixing the method name (=> `showWarningMessage`) and turn the `target` parameter across the various methods into `targetEl` ?

Reply:
Hello @bso-odoo, We could do that, but we decided to avoid unnecessary (out of task scope) differences as we received a few comments about that in the past. Let me know if we would like to proceed with that difference here.

cc: @divy-odoo

Reply:
Well... in this case I guess this is "kind of" in the scope - given that during this conversion we already rename the `$xyz` to `xyzEl`.
E.g. `target.classList. [...]` is an impacted line - and we are supposed to name those variables `*El` => it also impacts the method signature.
So, according to me, it can be done inside this PR.